modules/components/src/DataTable/DataTable.js

```diff
@@ -77,69 +79,72 @@ class DataTableWithToolbar extends React.Component {
 
   render() {
     const {
-      config,
-      fetchData,
-      setSelectedTableRows = () => {},
-      sqon,
-      selectedTableRows = null,
       allowTogglingColumns = true,
       allowTSVExport = true,
+      alwaysSorted = [],
+      columnDropdownText,
+      config,
       customActions = null,
+      customHeaderContent = null,
       data = null,
-      loading = null,
-      tableStyle,
-      toolbarStyle,
-      onFilterChange = () => {},
-      onColumnsChange = () => {},
-      onMultipleColumnsChange = () => {},
-      columnDropdownText,
       enableDropDownControls = false,
-      exportTSVText,
-      exportTSVFilename,
       exporter,
-      transformParams,
-      maxPagesOptions,
-      projectId = PROJECT_ID,
-      downloadUrl = urlJoin(ARRANGER_API, projectId, 'download'),
-      onSortedChange = () => {},
-      alwaysSorted = [],
+      exportTSVFilename,
+      exportTSVText,
+      fetchData,
+      filterInputPlaceholder,
       initalSelectedTableRows,
+      InputComponent,
       keepSelectedOnPageChange = false, // If false, this will reset the selection to empty on reload even if sessionStorage is enabled. To keep selections after reload set this to true.
+      loading = null,
+      maxPagesOptions,
+      onColumnsChange = () => {},
+      onFilterChange = () => {},
+      onMultipleColumnsChange = () => {},
+      onSortedChange = () => {},
+      projectId = PROJECT_ID,
+      selectedTableRows = [],
```

Comment:
Make the default for selectedTableRows an array instead of null.
Sort the other props alphabetically.